src/ucp/wireup/select.c

```diff
@@ -2326,8 +2326,10 @@ static ucs_status_t ucp_wireup_select_set_locality_flags(
     for (lane = 0; lane < key->num_lanes; ++lane) {
         rsc_index = key->lanes[lane].rsc_index;
         if ((rsc_index != UCP_NULL_RESOURCE) &&
-            (worker->context->tl_rscs[rsc_index].tl_rsc.dev_type ==
-             UCT_DEVICE_TYPE_SHM)) {
+            ((worker->context->tl_rscs[rsc_index].tl_rsc.dev_type ==
+             UCT_DEVICE_TYPE_SHM) &&
```

Comment:
do we really need to check UCT_DEVICE_TYPE_SHM, maybe enough to test UCT_IFACE_FLAG_INTER_NODE?

Reply:
removed the UCT_DEVICE_TYPE_SHM check here.